controllers/issues.js

```diff
@@ -10,9 +11,18 @@ const { SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
 
 const getIssues = async (req, res) => {
   try {
-    const { q: queryString } = req.query;
+    const { q: queryString, dev } = req.query;
     let issues = {};
-    if (queryString) {
+    const githubOrg = config.get("githubApi.org");
+    const githubIssuerUrlPattern = new RegExp(`^https://github.com/${githubOrg}/.+/issues/\\d+$`);
+
+    if (githubIssuerUrlPattern.test(queryString) && dev === "true") {
+      const url = new URL(queryString);
+      const issueUrlPaths = url.pathname.split("/");
+      const repositoryName = issueUrlPaths[2];
```

Comment:
what if issueUrlPaths has only 1 or 2 elements

Reply:
The ragex in [line 17](https://github.com/Real-Dev-Squad/website-backend/pull/1665/files/1d65902a4b30156b2cc817ed09e139aa4d57996f#diff-d0f9f68f392ba94b382d341a5f5651e2c8e06c368ff464d6d7d700bde9503273R17) will make sure we have required number of elements.